2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,24 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+
+let recipe1 = {
+  title: "Mole",
+  servings: 2,
+  ingredients: ["cinnamon", "cumin", "cocoa"]
+}
+
+console.log(recipe1.title);
+console.log(`Serves: ${recipe1.servings}`);
+console.log("Ingredients:");
+for (let ingredient of recipe1.ingredients) {
+  console.log(ingredient);
+}
+
+
```

Comment:
here we wanted you to create 5 receipts and show each key of each receipts  in one line , can you think of the solution, if you need help just @ me here, I can send you some tips and the solution :)